### PR TITLE
thousandeyes example for service response time monitoring

### PR DESCRIPTION
**Pull request by esara**
Tested locally with a sample response as:

I1012 23:00:33.473437   30789 thousandeyes.go:240] {"version":"v1","updateTime":1602558031,"scope":"Prometheus","source":"","topology":[{"uniqueId":"Datacloud","type":"service","name":"Datacloud","hostedOn":null,"matchIdentifiers":null,"partOf":null,"metrics":{"responseTime":[{"average":28,"max":29,"capacity":60000,"key":"https://datacloud.turbonomic.io/"}]}},{"uniqueId":"Turbonomic","type":"service","name":"Turbonomic","hostedOn":null,"matchIdentifiers":null,"partOf":null,"metrics":{"responseTime":[{"average":1,"max":2,"capacity":60000,"key":"https://turbonomic.com"}]}},{"uniqueId":"JIRA","type":"service","name":"JIRA","hostedOn":null,"matchIdentifiers":null,"partOf":null,"metrics":{"responseTime":[{"average":6,"max":8,"capacity":60000,"key":"https://my.atlassian.com"}]}},{"uniqueId":"Microsoft Office 365 Login","type":"service","name":"Microsoft Office 365 Login","hostedOn":null,"matchIdentifiers":null,"partOf":null,"metrics":{"responseTime":[{"average":12,"max":13,"capacity":60000,"key":"https://login.microsoftonline.com"}]}},{"uniqueId":"Salesforce NA57","type":"service","name":"Salesforce NA57","hostedOn":null,"matchIdentifiers":null,"partOf":null,"metrics":{"responseTime":[{"average":110,"max":111,"capacity":60000,"key":"https://na57.salesforce.com"}]}},{"uniqueId":"Slack","type":"service","name":"Slack","hostedOn":null,"matchIdentifiers":null,"partOf":null,"metrics":{"responseTime":[{"average":42,"max":42,"capacity":60000,"key":"http://api.slack.com"}]}}]}

### CODE REVIEW

File: example/thousandeyes/thousandeyes.go [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+				}
+				for _, metric := range metrics {
+					// Create service entity
+					service := dif.NewDIFEntity(test.TestName, "service")
+					capacity := float64(defaultCapacity)
+					service.AddMetrics("responseTime", []*dif.DIFMetricVal{{
+						Average:  &metric.AvgLatency,
+						Max:      &metric.MaxLatency,
+						Capacity: &capacity,
+						Key:      &url,
```
Comment (enlinxu) [inline example/thousandeyes/thousandeyes.go:223]: not so familiar with thousandeyes, but are we able to stitch with using url as the key?

**Comment by ading1977** [comment]
Thanks @esara for this! Did you test the metric server only, or test together with a DIF probe? If the latter, is the service stitched to anything? Or is it a standalone service?

File: example/thousandeyes/thousandeyes.go (hunk 1 above)
Comment (ading1977) [reply example/thousandeyes/thousandeyes.go:223]: Stitching needs to be done with `Matching("IP")` which I don't see in the code, so I assume this is sent as a standalone service without any stitching.

**Comment by esara** [comment]
> Thanks @esara for this! Did you test the metric server only, or test together with a DIF probe? If the latter, is the service stitched to anything? Or is it a standalone service?

I added a matchingip for the serverip of the service, which may stil stay standalone

**Pull request merged**